msteams-platform/graph-api/meeting-transcripts/meeting-insights.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Graph APIs to Fetch Meeting Insights
+description: Learn how to use the Meeting AI Insights API to fetch Teams meeting insights including conversation summaries, action items, and @mention utterances.
+ms.localizationpriority: high
+ms.topic: conceptual
+ms.owner: vichug
+ms.author: surbhigupta
+ms.date: 05/14/2025
+---
+
+# Get AI-generated meeting summaries with Meeting AI Insights API
+
+> [!IMPORTANT]
+>
+> * Meeting AI Insights API is available under the beta version in Microsoft Graph. For more information, see [versioning, support, and breaking change policies for Microsoft Graph](/graph/versioning-and-support#beta-version).
+> * Meeting AI Insights API is part of the Microsoft 365 Copilot API namespace. Insights can only be fetched on behalf of a Microsoft 365 Copilot licensed user. For more information, see [license requirements for Meeting Insights API](/graph/teams-licenses#license-requirements-for-teams-meeting-ai-insights-apis).
+
+The Meeting AI Insights API enables you to programmatically access structured AI-generated insights from transcribed Microsoft Teams meetings. These insights include:
+
+* Comprehensive conversation summaries
+* Action items extracted from the discussion
+* Participant-specific @mention utterances
+
+The API empowers you to deliver intelligent meeting experiences, such as surfacing key takeaways or generating follow-ups, without the need to build or maintain your own AI infrastructure. Insights are generated after the meeting concludes and are accessible through Microsoft Graph API endpoints.
+
+The API provides AI-generated insights for all meetings and calls for which a recap is available, including scheduled private and channel meetings, one-on-one and group Voice over Internet Protocol (VoIP) calls, webinars, and adhoc meetings.
+
+> [!NOTE]
+> Meeting AI Insights API is only available for organizers and presenters of a town hall meeting.
+
+## Use cases
```

Comment:
Should we mention that AI Insights generation is triggered by the user from Teams UI (on-demand) or in 4-hours and only then these APIs will return values? 

For diagnosing issues, do we have a section? It would be good to confirm that AI Insights are available in Teams if the API does not return values because of the above delay.

Reply:
I think that might be too much detail for these docs. We've added the 4 hour delay. Even if the client has the recap, our API may still not return the results so cannot provide that in the diagnostic section as well